src/Apps/Artist/Routes/Overview/index.tsx

```diff
@@ -122,24 +178,49 @@ export const OverviewRouteFragmentContainer = createFragmentContainer(
         partner_shows
       }
 
+      href
+      is_consignable
+
+      # NOTE: The following are used to determine whether sections
+      # should be rendered.
+
       biography_blurb(format: HTML, partner_bio: true) {
         text
         credit
       }
 
-      href
-      is_consignable
+      currentEvent {
+        name
+      }
 
-      ...Genes_artist
+      related {
```

Comment:
i see, so you basically duplicate the fragments which return the actual content to each component (or in a slimmed down way, just plucking the minimal `id`), as an exist-y check.

I think this is 👍 given the current patterns and metholodgy, but it'd be great for it to be less...crufty! One possibility is to move these calls into Metaphysics and enhance `statuses` or something, which might be nice to share with other clients possibly. But that can also always be done in the future if needed.

Reply:
Yah, moving to MP actually makes way more sense -- this duplication was bothering me and I tried all different ways to get it to return properly. Will give it 👀

Reply:
@mzikherman - I'm gonna merge to get these issues out and add a FIXME and lets pair tomorrow on this from a MP perspective, I'm curious about some approaches.